presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,83 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix disaggregated coordinator resource group aggregation issue related to ``WAITING_FOR_PREREQUISITE`` queries.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory over-counting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in :func:`tdigest_agg`.
+* Add :func:`to_base32` and :func:`from_base32` varbinary functions.
+* Add support for ``TRUNCATE TABLE`` statement.
+* Add support to set timeout on the query analyzer runtime to prevent long running query analysis like complex regular expressions. The timeout duration can be set by the configuration property ``planner.query-analyzer-timeout` or session property ``query_analyzer_timeout``.
```

Comment:
```suggestion
* Add support to set timeout on the query analyzer runtime to prevent long running query analysis like complex regular expressions. The timeout duration can be set by the configuration property ``planner.query-analyzer-timeout`` or session property ``query_analyzer_timeout``.
```

Reply:
corrected